docs/index.md

```diff
@@ -111,28 +107,56 @@ cargo test
 If you have any issues building because of the dependency on `rattler` checkout
 its [compile steps](https://github.com/mamba-org/rattler/tree/main#give-it-a-try).
 
-## Update
-
-Updating is as simple as installing, rerunning the installation script gets you the latest version.
+### Installer script options
 
 === "Linux & macOS"
+
+    The installation script has several options that can be manipulated through environment variables.
+
+    | Variable             | Description                                                                        | Default Value         |
+    |----------------------|------------------------------------------------------------------------------------|-----------------------|
+    | `VERSION`            | The version of pixi getting installed, can be used to up- or down-grade.           | `latest`              |
+    | `PIXI_HOME`          | The location of the binary folder.                                                  | `$HOME/.pixi`         |
+    | `PIXI_ARCH`          | The architecture the pixi version was built for.                                    | `uname -m`            |
+    | `PIXI_NO_PATH_UPDATE`| If set the `$PATH` will not be updated to add `pixi` to it.                         |                       |
+    | `TMP_DIR`            | The temporary directory the script uses to download to and unpack the binary from.   | `/tmp`                |
```

Comment:
not that it matters 🤔 

![image](https://github.com/prefix-dev/pixi/assets/29506042/eb695e83-bbe9-4527-86db-aef470fe739c)
```suggestion
    | `VERSION`            | The version of pixi getting installed, can be used to up- or down-grade.           | `latest`              |
    | `PIXI_HOME`          | The location of the binary folder.                                                 | `$HOME/.pixi`         |
    | `PIXI_ARCH`          | The architecture the pixi version was built for.                                   | `uname -m`            |
    | `PIXI_NO_PATH_UPDATE`| If set the `$PATH` will not be updated to add `pixi` to it.                        |                       |
    | `TMP_DIR`            | The temporary directory the script uses to download to and unpack the binary from. | `/tmp`                |
```